### ISSUE TITLE
Constantly calling skip overlay check

### ISSUE THREAD

**Issue by smkiewel** [mtgatracker/mtgatracker#422]
4.5.4 has some bug that makes the skip overlay check happen constantly. See the hundreds of console.log statements.
![skip-overlay](https://user-images.githubusercontent.com/745723/48624031-7222d480-e979-11e8-8b43-e523c280b26f.PNG)

**Comment by smkiewel** [comment]
... and now I can't replicate it.

**Comment by shawkinsl** [comment]
I don't think this is a bug (unless you are certain you've never disabled "Overlay only on MTGA." Maybe the debug log is excessive, but this function is on a setInterval of 200ms.
